universal-application-tool-0.0.1/app/views/admin/AdminLayout.java

```diff
@@ -73,15 +87,24 @@ private ContainerTag renderNavBar() {
             .with(span("Civi"), span("Form").withClasses(Styles.FONT_THIN));
 
     ContainerTag adminHeader =
-        nav()
-            .with(headerIcon, headerTitle)
-            .with(headerLink("Questions", questionLink))
-            .with(headerLink("Programs", programLink))
-            .with(headerLink("Versions", versionLink))
-            .with(headerLink("Intermediaries", intermediaryLink))
-            .with(headerLink("Logout", logoutLink, Styles.FLOAT_RIGHT))
-            .withClasses(AdminStyles.NAV_STYLES);
-    return adminHeader;
+        nav().with(headerIcon, headerTitle).withClasses(AdminStyles.NAV_STYLES);
+
+    // Don't include nav links for program admin.
+    if (adminType.equals(AdminType.PROGRAM_ADMIN)) {
+      return adminHeader.with(headerLink("Logout", logoutLink, Styles.FLOAT_RIGHT));
+    }
+
+    String questionLink = controllers.admin.routes.AdminQuestionController.index().url();
+    String programLink = controllers.admin.routes.AdminProgramController.index().url();
+    String versionLink = routes.AdminVersionController.index().url();
+    String intermediaryLink = routes.TrustedIntermediaryManagementController.index().url();
+
+    return adminHeader
+        .with(headerLink("Questions", questionLink))
```

Comment:
Suggestion (non-blocking): while you are at it, would you move program to the first link since it seems more natural that way in terms of user flow (also according to recent admin UX mock)

Reply:
Good call!